.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       osx_64_python3.13.____cp313:
         CONFIG: osx_64_python3.13.____cp313
         UPLOAD_PACKAGES: 'True'
+      osx_64_python3.9.____cpython:
+        CONFIG: osx_64_python3.9.____cpython
+        UPLOAD_PACKAGES: 'True'
```

Comment:
@xhochy how can we remove python 3.9?

Reply:
```
build:
    skip: true  # [py<39]
```

Reply:
is that documented somewhere?

Reply:
btw where can we add it?

Reply:
https://conda-forge.org/docs/maintainer/adding_pkgs/#skipping-builds
https://github.com/conda-forge/mlflow-feedstock/blob/main/recipe/meta.yaml#L14